Open_Science_New.html

```diff
@@ -69,93 +65,136 @@ <h4 style="padding-bottom:0px; margin-bottom:0px; text-shadow:-1px -1px 0 #000,
+        <section data-transition="fade" data-background="images/data_publishing.png">
+	      <h1 style="text-shadow:-1px -1px 0 #000,1px -1px 0 #000,-1px 1px 0 #000,1px 1px 0 #000;"><font color="indigo">Data Publishing<br>
+		- Not Data Sharing -</font></h1>
+            <p style="text-shadow:-1px -1px 0 #000,1px -1px 0 #000,-1px 1px 0 #000,1px 1px 0 #000;">This is an important differentiation. Data publishing calls for extensive reviewing and quality control processes.</p>
+	  </section>
+
+	  <!-- EU going Open Access -->
+        <section>
+	      <table class="reveal">
+		<tr>
+		  <td width="50%" style="vertical-align: middle">
+		    <a href="https://www.theguardian.com/science/2016/may/28/eu-ministers-2020-target-free-access-scientific-papers" target="_blank"><img src="images/Open_access.png"></a>
+		  </td>
+
+		  <td width="50%" style="vertical-align: middle;">
+		    <a href="https://www.scienceeurope.org/" target="_blank"><img src="images/ScienceEurope.png" style="border:0; border-radius:30px;"></a>
+		    <p><small><br>This intiative was officially launched as<br>
+                "Plan S" by Brussels-based Science Europe in September of 2018.</small></p>
+		  </td>
+
+	</tr></table>
+	    </section>
+
+	  <!-- FAIR -->
+	  <section data-transition="fade">
+	    <a href="https://www.nature.com/articles/sdata201618"><img src="images/FAIR.png"></a>
+	  </section>
+
+	  <!-- COBIDAS -->
+	  <section data-transition="fade">
+          <table class="reveal">
+		<tr>
+		  <td width="50%" style="vertical-align: middle">
+		    <a href="https://www.nature.com/articles/nn.4500"><img src="images/COBIDAS_Nature_new.png"></a>
+		  </td>
+
+		  <td width="50%" style="vertical-align: middle;">
+		    <p>Best practices and standards are important in Open Science for usability and reproducability.</p>
+		  </td>
+	</tr></table>
+	  </section>
+
+        <!-- WHAT'S INVOLVED? -->
+   	<section>
+	  <h2><font color="12a3d8">What’s involved?</font></h2>
+	  <p>
+	    Longitudinal Acquisition, Storage and Curation, <font color="f27c30">Interoperability, Reproducibility</font>, Transfer, Anonymization, Security, Privacy, Ethics, APIs, Validation, Quality Control, Protocol Checking, Preprocessing, Analysis, HPC, Provenance, Ontological Standarization,<br><font color="f27c30">Data Harmonization</font>, Upgrades, Maintenance, Bug Fixes, User Interface, Javascript, Bootstrap, Tracking, <font color="f27c30">Extensibility</font>, Data Management, Summary Statistics, Workflows, Development, Tool Integration, <font color="f27c30">Data Sharing</font>, Download, Multi-Modal Linking, <font color="f27c30">Querying</font>, Image Processing, Visualization, Networking, System Administration, Partnerships, Funding, HR <font color="f27c30">...No big deal!</font>
+          </p>
+        </section>
+        </section>
+
+          <!-- Open Science Initiatives - Column 2 -->
+    <section data-transition="convex">
 	    <h2>Open Science Initiatives</h2>
```

Comment:
@christinerogers there's text hidden below these images that doesn't fit on the page - do you know a fix?

Reply:
https://htmlpreview.github.io/?https://github.com/knoll-alex/samirdas.github.io/blob/ak-Open_Science/Open_Science_New.html#/2